contributors/devel/e2e-tests.md

```diff
@@ -576,7 +576,7 @@ suite, it receives a `[Feature:.+]` label, e.g. `[Feature:Performance]` or
 `[Feature:Ingress]`. `[Feature:.+]` tests are not run in our core suites,
 instead running in custom suites. If a feature is experimental or alpha and is
 not enabled by default due to being incomplete or potentially subject to
-breaking changes, it does *not* block the merge-queue, and thus should run in
+breaking changes, it does *not* block the merge pool, and thus should run in
```

Comment:
this might be confusing without more background on pools

Reply:
It does _not_ block PR merges